2018-19/B12 Create CHI - LTC Lookup.sps

```diff
@@ -98,10 +98,11 @@ Else.
     End if.
 End if.
 
-Temporary.
-select if duplicate_chi = 1 or duplicate_chi_postcode = 1.
+If duplicate_chi = 1 or duplicate_chi_postcode = 1 Error = 1.
+Frequencies Error.
```

Comment:
Could we change this to `crosstabs Error by chi.` that way if there is an error we will also get the duplicate CHI(s) printed in the output.